pkg/controller/queuejobresources/utils.go

```diff
@@ -74,41 +78,41 @@ func PendingPodsFailedSchd(pods []*v1.Pod) map[string][]v1.PodCondition {
 
 // filterPods returns pods based on their phase.
 func GetPodResourcesByPhase(phase v1.PodPhase, pods []*v1.Pod) *clusterstateapi.Resource {
-    req := clusterstateapi.EmptyResource()
-    for i := range pods {
-        if  pods[i].Status.Phase == phase {
-            for _, c := range pods[i].Spec.Containers {
-                req.Add(clusterstateapi.NewResource(c.Resources.Requests))
-            }
-        }
-    }
-    return req
+	req := clusterstateapi.EmptyResource()
```

Comment:
Shouldn't we fix this intendation to be 4 spaces? The other function in this file (i.e., `PendingPodsFailedSchd`) doesn't have 8 spaces, if the point is to be consistent.

Reply:
I think that some point after we merge the quota management branch, we should format the entire code base using: https://go.dev/blog/gofmt and add a rule to the make file to run the formatter and perhaps go vet only on changed file.